src/humanpy/hum_kin1.py

```diff
@@ -0,0 +1,263 @@
+PACKAGE = 'humanpy'
```

Comment:
Also each function needs a function comment that explains args, outputs, general function, etc.